config/environments/production.rb

```diff
@@ -44,7 +49,7 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
-  config.log_level = :info
+  config.log_level = :debug
```

Comment:
The update task changed this, seems fine to me.

Reply:
It's much more verbose now, which has been somewhat useful in some debugging, but also much noisier.  I think probably best to turn this back down to :info after the holiday.